pkg/settings/agent_customization.go

```diff
@@ -54,7 +54,7 @@ const (
         "preference": {
           "matchExpressions": [
             {
-              "key": "node-role.kubernetes.io/master",
+              "key": "node-role.kubernetes.io/control-plane",
```

Comment:
`"node-role.kubernetes.io/control-plane"` is already present above, better to clean this entry?

Reply:
oh, good catch, Thanks! Totally missed that duplication in my late-night haze 🫠

Reply:
This was marked as resolved but was not resolved.